### ISSUE TITLE
Line break in version 3

### ISSUE THREAD

**Issue by xrjunque** [mathjax/MathJax#3230]
Version 3 does not account line breaks. When will it be available?
Or is this postposed as a feature for a future release?

Thanks in advance

**Comment by dpvc** [comment]
Line breaking support is in v4 (currently in beta release).  See the [release notes](https://github.com/mathjax/MathJax-src/releases/tag/4.0.0-alpha.1#linebreaking) from the original alpha release that included it.

**Comment by dpvc** [comment]
This is a duplicate of #2312.

**Comment by xrjunque** [comment]
Yes, I read that thread before, but 9 years have gone by and I thought there could be any news since then.
________________________________
De: Davide P. Cervone ***@***.***>
Enviat el: dimarts, 14 de maig de 2024 16:11
Per a: mathjax/MathJax ***@***.***>
A/c: Xavier Ramon Junqué i de Fortuny ***@***.***>; Author ***@***.***>
Tema: Re: [mathjax/MathJax] Line break in version 3 (Issue #3230)


This is a duplicate of #2312<https://github.com/mathjax/MathJax/issues/2312>.

**Comment by dpvc** [comment]
I think you have miscalculated the time; it is only 4 and a half years (the linked issue was opened in October of 2019).  Also, line breaking has been in the v4 alpha and beta releases since October of 2022 (two years after #2312), as indicated by several my comments in that issue (e.g., [this one](https://github.com/mathjax/MathJax/issues/2312#issuecomment-1649658644)).

In any case, my marking this as a duplicate was not meant as a criticism, but was for bookkeeping purposes to help organize the issues that are open, and keep the related ones linked for cross reference.

**Comment by xrjunque** [comment]
Well, in my current application I leave open to the user to employ 2.7.2 version prior to version 3; or version 3 as it is -or I think so- the last stable version.
Any way, it would be nice to have translated MathML's   <mspace linebreak="newline" />  by MathJax version 3.
Please pardon me, if I don't know what difficulties this can represent to you.

________________________________
De: Davide P. Cervone ***@***.***>
Enviat el: dimarts, 14 de maig de 2024 16:27
Per a: mathjax/MathJax ***@***.***>
A/c: Xavier Ramon Junqué i de Fortuny ***@***.***>; Author ***@***.***>
Tema: Re: [mathjax/MathJax] Line break in version 3 (Issue #3230)


I think you have miscalculated the time; it is only 4 and a half years (the linked issue was opened in October of 2019). Also, line breaking has been in the v4 alpha and beta releases since October of 2022 (two years after #2312<https://github.com/mathjax/MathJax/issues/2312>), as indicated by several my comments in that issue (e.g., this one<https://github.com/mathjax/MathJax/issues/2312#issuecomment-1649658644>).

In any case, my marking this as a duplicate was not meant as a criticism, but was for bookkeeping purposes to help organize the issues that are open, and keep the related ones linked for cross reference.
